fearless/Modules/Staking/ControllerAccount/ControllerAccountInteractor.swift

```diff
@@ -0,0 +1,111 @@
+import UIKit
+import SoraKeystore
+import RobinHood
+
+final class ControllerAccountInteractor {
+    weak var presenter: ControllerAccountInteractorOutputProtocol!
+
+    let singleValueProviderFactory: SingleValueProviderFactoryProtocol
+    let substrateProviderFactory: SubstrateDataProviderFactoryProtocol
+    let runtimeService: RuntimeCodingServiceProtocol
+    private let selectedAccountAddress: AccountAddress
+    private let accountRepository: AnyDataProviderRepository<AccountItem>
+    private let operationManager: OperationManagerProtocol
+    private let feeProxy: ExtrinsicFeeProxyProtocol
+    private let extrinsicService: ExtrinsicServiceProtocol
+    private lazy var callFactory = SubstrateCallFactory()
+
+    private var stashItemProvider: StreamableProvider<StashItem>?
+    private var accountInfoProvider: AnyDataProvider<DecodedAccountInfo>?
+
+    init(
+        singleValueProviderFactory: SingleValueProviderFactoryProtocol,
+        substrateProviderFactory: SubstrateDataProviderFactoryProtocol,
+        runtimeService: RuntimeCodingServiceProtocol,
+        selectedAccountAddress: AccountAddress,
+        accountRepository: AnyDataProviderRepository<AccountItem>,
+        operationManager: OperationManagerProtocol,
+        feeProxy: ExtrinsicFeeProxyProtocol,
+        extrinsicService: ExtrinsicServiceProtocol
```

Comment:
I think we can initialise service at this point because we don't know which account is the stash